src/cpp/session/modules/SessionFind.cpp

```diff
@@ -240,6 +403,15 @@ class GrepOperation : public boost::enable_shared_from_this<GrepOperation>
    }
 
 private:
+   struct LineInfo
+   {
+      std::string leftContents;
+      std::string rightContents;
```

Comment:
decodedPreview and decodedContents trim the leading and trailing contents of the line when they're set to avoid tabs and spaces. We need to save these contents so they can be used to create the encodedContents which is what actually gets printed back to the file. Do you think the names leadingContents and trailingContents would be more intuitive?

Reply:
Yeah, that might be clearer to me, maybe with some mention of them being whitespace contents? Boost uses `Ws` as shorthand for whitespace, so maybe that would be obvious without making the name too long?

Possible nit I just noticed: do we normally leave out the trailing _ for struct members?

Reply:
I changed these to leadingWhitespace and trailingWhitespace since the word Contents wasn't telling you much. Looking around the code, it doesn't appear we use the trailing _ in structs